packages/sitecore-jss-nextjs/src/components/Link.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactElement } from 'react';
+import NextLink from 'next/link';
+
+export interface LinkFieldValue {
+  href?: string;
+  className?: string;
+  title?: string;
+  target?: string;
+  editable?: string;
+  [attributeName: string]: any;
+}
+
+export interface LinkField {
+  value?: LinkFieldValue;
+  editableFirstPart?: string;
+  editableLastPart?: string;
+}
+
+export type LinkProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  /** The link field data. */
+  field: LinkField | LinkFieldValue;
+  /**
+   * Can be used to explicitly disable inline editing.
+   * If true and `field.editable` has a value, then `field.editable` will be processed and rendered as component output. If false, `field.editable` value will be ignored and not rendered.
+   * @default true
+   */
+  editable?: boolean;
+
+  /**
+   * Displays a link text ('description' in Sitecore) even when children exist
+   * NOTE: when in Sitecore Experience Editor, this setting is ignored due to technical limitations, and the description is always rendered.
+   */
+  showLinkTextWithChildrenPresent?: boolean;
+
+  /**
+   * Should component prefetch route
+   */
+  prefetch?: boolean;
+
+  /**
+   * Locale which will be used when prefetch route
+   */
+  locale?: string;
```

Comment:
@ambrauer Please, look at this article, if I will not provide locale, I will load route in defaultLocale (en)
https://nextjs.org/docs/advanced-features/i18n-routing#transition-between-locales

Reply:
Agree, removed `prefetch` and added `locale={false}` according to this article: https://nextjs.org/docs/advanced-features/i18n-routing#transition-between-locales
I provided `locale` prop - because by default it loaded all data using current locale - `en`. Now it will detect locale from `href` of link

Reply:
Ahh, cool. `locale={false}` is exactly what we'd want.